src/commands/schema/commit.mjs

```diff
@@ -73,12 +73,12 @@ function buildCommitCommand(yargs) {
     })
     .example([["$0 schema commit"]])
     .version(false)
-    .help("help", "show help");
+    .help("help", "Show help.");
```

Comment:
I can't comment on the correct line (it's context/not changed), but the string on line 69 of this file should end with a period:
```
description: "Prompt for user input (e.g., confirmations)",
```

Reply:
Updated with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a) and [`a257cb8` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/a257cb8ddfbc1003d0ccf0af2a5cc772f2c361a4)